fax/constrained/constrained.py

```diff
@@ -225,7 +225,8 @@ def lagrange_update(i, grads, opt_state, *args, **kwargs):
             An new packed optimization state with the updated parameters and
             Lagrange multipliers.
         """
-        grads = (grads[0], tree_util.tree_map(lax.neg, grads[1]))
+        params_grad, multipliers_grad = grads
```

Comment:
Just aesthetical?

Reply:
It implicitly enforces the `grads` structure as the docstring requires
in the indexing version `grads = [1, 2, 3]` would be acceptable, now it fails.
I'm removing it nonetheless because it's not an extragrad change